config/kubevirtci/kustomization.yaml

```diff
@@ -1,6 +1,20 @@
-namespace: kube-system
+namespace: kvcluster
 bases: 
  - ../base
 
 patchesStrategicMerge:
   - manager_image_patch.yaml
+
+patchesJson6902:
```

Comment:
Could we even add a ownerreference to the kccm? So that when kccm is deleted, this Rolebinding is deleted too?

Reply:
that's a good one I will try.